website/spam_detection.py

```
@@ -0,0 +1,83 @@
+             dict: {
+                 'is_spam': bool,
+                 'spam_score': int (0-100),
+                 'reason': str
+             }
+        """
+        if not self.api_key:
+            return {"is_spam": False, "spam_score": 0, "reason": "Spam detection not available"}
+        try:
+            prompt = self._get_system_prompt(title, description, url)
+            response = self.get_gemini_response(prompt)
+            return response
```

Ruff (0.14.4)

Consider moving this statement to an `else` block

(TRY300)